tibiawikisql/__main__.py

```diff
@@ -2,12 +2,12 @@
 import time
 
 from tibiawikisql.utils import database, fetch_deprecated_list, spells, items, npcs, achievements, houses, quests, \
-    creatures, map, imbuements
+    creatures, map, imbuements, charms
 
 DATABASE_FILE = "tibia_database.db"
 SKIP_IMAGES = "skipimages" in sys.argv
 
-__version__ = "1.1.1"
+__version__ = "1.1.2"
```

Comment:
Usually this shouldn't be done on the feature branch. Next version would be 1.2.0 anyway, since a new feature is being added.